scripts/spack/configs/toss_3_x86_64_ib/compilers.yaml

```diff
@@ -3,32 +3,32 @@ compilers:
     environment: {}
     extra_rpaths: []
     flags:
-      cflags: --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.1.0
-      cxxflags: --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.1.0
+      cflags: --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1
+      cxxflags: --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1
     modules:
     - clang/9.0.0
     operating_system: rhel7
     paths:
       cc: /usr/tce/packages/clang/clang-9.0.0/bin/clang
       cxx: /usr/tce/packages/clang/clang-9.0.0/bin/clang++
-      f77: /usr/tce/packages/gcc/gcc-8.1.0/bin/gfortran 
-      fc: /usr/tce/packages/gcc/gcc-8.1.0/bin/gfortran
+      f77: /usr/tce/packages/gcc/gcc-8.3.1/bin/gfortran
+      fc: /usr/tce/packages/gcc/gcc-8.3.1/bin/gfortran
     spec: clang@9.0.0
     target: x86_64
 - compiler:
     environment: {}
     extra_rpaths: []
     flags:
-      cflags: --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.1.0
-      cxxflags: --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.1.0
+      cflags: --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1
+      cxxflags: --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.3.1
```

Comment:
Sounds like a good idea, unless there's a specific reason we're using `gcc@8.1.0` w/ `intel@19` instead of `gcc@8.3.1`.
(and if so, it would be good to document that).

Since the TPLs can take a bit to rebuild, perhaps it's something we can upgrade in the future though.

Reply:
We really should do a full evaluation of what compilers we are building. I have a feeling we have drifted completely away from what our customers are using.

I agree though I should have bumped it across the board.